src/csharp/Microsoft.Spark/Sql/DataFrameReader.cs

```diff
@@ -255,6 +255,15 @@ public DataFrameReader Options(Dictionary<string, string> options)
         /// <param name="paths">Input paths</param>
         /// <returns>DataFrame object</returns>
         public DataFrame Orc(params string[] paths) => LoadSource("orc", paths);
+        
+        /// <summary>
+        /// Returns the specified table as a DataFrame. This is a table that is configured in the
+        /// Hive metabase.
```

Comment:
Is this always true?  This only works with tables that are configured in hive ?

Reply:
I've removed it now